app/services/InMemoryTrafficLightService.scala

```diff
@@ -1,19 +1,17 @@
 package services
-import javax.inject.Inject
-import model.TrafficLight.ongoingRequests
+import javax.inject.Singleton
 import model.{Color, TrafficLight}
-import play.api.cache.AsyncCacheApi
-import play.api.libs.json._
-import play.api.libs.ws._
-import play.api.mvc._
-import play.api.cache._
-import scala.language.postfixOps
+
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
-import scala.concurrent.duration.DurationInt
+import scala.language.postfixOps
+
+@Singleton
```

Comment:
We definitely need it to be `@Singleton` because we only want _one_ `TrafficLightService`. If we would have multiple, not all clients would see the same state (each client might have their own version of reality).